pkg/client/cluster_config.go

```diff
@@ -39,6 +39,6 @@ func NewClusterConfig(cfg *rest.Config) *rest.Config {
 // cluster endpoint.
 func ConfigWithCluster(cfg *rest.Config, clusterName logicalcluster.Name) *rest.Config {
 	r := rest.CopyConfig(cfg)
-	r.Host = r.Host + "/" + clusterName.Path()
+	r.Host = r.Host + clusterName.Path()
```

Comment:
Maybe check if r.Host ends with `/` and only include it if it's not there?

Reply:
`clustername.Path()` always has a `/` in front, so even if `r.Host()` does not have it should be fine. Ref: https://github.com/kcp-dev/logicalcluster/blob/b0a45e2ca1e3567bbe4f559a458dbdcfa04cdace/name.go#L71

Reply:
👍